test/templates/overlay/index.njk

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.njk" %}
+
+{% block content %}
+    <script>
+    function toggle(type, show) {
+        var overlay = document.querySelector('.fd-overlay--' + type);
+        if (overlay) {
+            overlay.setAttribute('aria-hidden', (show ? "false" : "true"));
```

Comment:
Are these still styled based on the `aria-hidden` attribute?

Reply:
They are, yes, but in the source sass file, there was no class-based alternative.  I can look into adding an `is-hidden` class to it.

Reply:
I think if this is something doing for all other "self-contained" components, this is the time to do it for `overlay` too.

Reply:
^Agree with Mark for consistency